core/src/main/java/com/novoda/noplayer/internal/exoplayer/SimpleRenderersFactory.java

```diff
@@ -165,6 +167,7 @@ private void buildVideoRenderers(Context context,
                 allowedVideoJoiningTimeMs,
                 drmSessionManager,
                 DO_NOT_PLAY_CLEAR_SAMPLES_WITHOUT_KEYS,
+                ENABLE_DECODER_FALLBACK,
```

Comment:
> This may result in using a decoder that is slower/less efficient than the primary decoder.

I'm wondering if we should be attempting to log stuff like this when it happens. Slower or less efficient may lead to problems during playback.

Reply:
Isn't this a replacement for `MediaCodecSelector.DEFAULT_WITH_FALLBACK`? We provide a flag to use that on the builder so this should be in sync with that value

Reply:
Yes. My point was more about the effect that this could have in general regardless of how it was implemented. 

If we use a fallback decoder then we could potentially have playback issues like stuttering, because we are using a less efficient decoder. It was a thought, just thinking out loud 😄

Reply:
Yes, that should be a separate issue that we can look into regardless of this change.

My concern was more about the fact that this seems to be changing the behavior in an unexpected way since it will always enable the fallback regardless of the value that the clients use in the builder.

Reply:
I guess you are referring to https://github.com/novoda/no-player/blob/master/core/src/main/java/com/novoda/noplayer/PlayerBuilder.java#L102? This is not the same thing 😄 the one in the builder refers to force downgrading to use an insecure decoder when we are able to. The one changed in this PR is going to use a fallback decoder when we fail to initialise the one we want.